x-pack/plugins/security_solution/public/entity_analytics/components/entity_store/hooks/use_entity_store.ts

```diff
@@ -95,7 +102,11 @@ export const useStopEntityEngineMutation = (options?: UseMutationOptions<{}>) =>
         timestamp: new Date().toISOString(),
         action: 'stop',
       });
-      return Promise.all([stopEntityEngine('user'), stopEntityEngine('host')]);
+      return Promise.all(
```

Comment:
Unlike the init mutation, we don't have a route to start/stop/delete all engines, so here we can keep this logic, although I suggest renaming the hook, or adding the entity types as parameters

Reply:
I added the changes here: https://github.com/elastic/kibana/pull/202344/commits/6943a53c05c50863c21f285c3ab06fc61548f3bf